bin/man_page_gen.ml

```diff
@@ -0,0 +1,26 @@
+open Stdune
+
+let cmdliner_cmd = Dune_cmd.cmd
+
+let write_man_page (cmd : string list) =
+  let dash_seperated = String.concat ~sep:"-" cmd in
+  let oc = open_out (dash_seperated ^ ".1") in
+  cmd @ [ "--help=plain" ] |> Array.of_list |> fun argv ->
+  let _ =
```

Comment:
we add type annotations to ignored values. btw, we should update the hacking guide to mention this.